README.md

```diff
@@ -18,23 +18,11 @@ This oracle handles the following cases among others (let's say that our pair is
 - A/C and C/B are feeds (typically, WBTC/BTC and BTC/ETH).
 - A'/C and B/C are feeds, and there is an exchange rate between A and A'. (typically A=sDAI and A'=DAI).
 
-## WstETH/stETH Exchange Rate Adapter
-
-A specific implementation, the `WstEthStEthExchangeRateChainlinkAdapter`, provides the exchange rate between wstETH and stETH as a Chainlink-interface-compliant feed.
-
-This adapter is deployed on Ethereum Mainnet at the address [0x905b7dAbCD3Ce6B792D874e303D336424Cdb1421](https://etherscan.io/address/0x905b7dabcd3ce6b792d874e303d336424cdb1421#code).
+### MorphoChainlinkOracleV2Factory
 
-## MorphoChainlinkOracleV2Factory
-
-The `MorphoChainlinkOracleV2Factory` is a factory that deploys `MorphoChainlinkOracleV2` instances.
-
-The factory facilitates the creation and indexing of `MorphoChainlinkOracleV2` oracles. This factory contract streamlines the process of deploying new oracles, catering to various market pairs without the need to deploy individual contracts manually for each pair.
-
-### Usage
-
-The Ethereum Mainnet address of this factory is [0x3A7bB36Ee3f3eE32A60e9f2b33c1e5f2E83ad766](https://etherscan.io/address/0x3a7bb36ee3f3ee32a60e9f2b33c1e5f2e83ad766#code).
+#### Deploy an Oracle
 
-To create a new `MorphoChainlinkOracleV2` oracle using the factory, call the `createMorphoChainlinkOracleV2` function with the following parameters:
+To deploy a `MorphoChainlinkOracleV2` on the Ethereum mainnet, use the `MorphoChainlinkOracleV2Factory` to create a new oracle. Call the `createMorphoChainlinkOracleV2` function with the following parameters:
```

Comment:
Let's put every sentence in a new line. It's easier that way to see the diff when doing big changes